<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reserv_add</class>
 <widget class="QDialog" name="reserv_add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>에약 등록</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 20pt &quot;ori1Uni&quot;;</string>
   </property>
   <property name="text">
    <string>렌터카 예약</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>290</y>
     <width>651</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>전체</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>휘발유</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>경유</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LPG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>전기</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="booking_btn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>30</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius: 5px;</string>
   </property>
   <property name="text">
    <string>예약</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="reserv_time">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>194</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="return_time">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>194</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>67</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>예약시간</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>67</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>반납시간</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>고객 선택</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>차량 선택</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="common_btn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>일반</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="premium_btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>150</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>premium</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="vip_btn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>150</y>
     <width>112</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>vip</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="usertable">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>150</y>
     <width>251</width>
     <height>131</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
